sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/DateExpressionsSuite.scala

```diff
@@ -1146,4 +1146,49 @@ class DateExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
           Literal("yyyy-MM-dd'T'HH:mm:ss.SSSz")), "Fail to parse")
     }
   }
+
+  test("SPARK-31710:Fix millisecond and microsecond convert to timestamp in to_timestamp") {
+    withSQLConf() {
+      checkEvaluation(
+        GetTimestamp(
```

Comment:
I thought we are going to add 2 new functions `TIMESTAMP_MILLIS` and `TIMESTAMP_MICROS`, to follow big query. Why do we overload `GetTimestamp`?

Reply:
I think your idea is right, I will add the three functions later and make a revert.